### ISSUE TITLE
App install fails under Owncloud 9.1.5 (Assume Nextcloud is still a drop in replacement)

### ISSUE THREAD

**Issue by tschiu** [nextcloud/tasks#81]
### Steps to reproduce
1.  Assume that this app is still supposed to work under Owncloud 9.1.5 :-)
2.  Install Owncloud 9.1.5
3.  cd /var/www/owncloud/apps
4.  git clone https://github.com/nextcloud/tasks

### Expected behaviour
Under the assumption that Nextcloud and owncloud haven't diverged yet:
When activating the app from the Apps menu, the app should work

### Actual behaviour
After cloning, accessing results in a page stating that an update is required:
"App update required.  These apps will be updated:  Tasks (tasks)
Please make sure that the database, the config folder, and the data folder have been backed up before proceeding...."
![tasks after cloning](https://cloud.githubusercontent.com/assets/16103278/26449605/1dc7e774-4121-11e7-8710-dc8c2688244c.JPG)
Click "Start update"
Next screen states "Updating to 9.1.5.  Detailed logs..."
Gets up to 5/24: Checking table oc_filecache
![in-process update](https://cloud.githubusercontent.com/assets/16103278/26449756/08e1cbd0-4122-11e7-8f35-c43a84804255.JPG)

Then shows an error message
![error occurred](https://cloud.githubusercontent.com/assets/16103278/26450138/ee7657f0-4123-11e7-9261-a06fad34c614.JPG)
cd /var/www/owncloud/apps
sudo mv tasks <somewhere else>
sudo service apache2 restart
sudo -u www-data php occ maintenance:mode --off
gets us back to a functional system

### Server configuration

**Operating system**:  
Ubuntu 16.04.2

**Web server:**  
Apache2

**Database:**  
MySQL 5.1.6

**PHP version:** 
PHP 7.0.15-0ubuntu0.16.04.4 (cli) ( NTS )

**Nextcloud version:** (see Nextcloud admin page)
ownCloud version 9.1.5

**Tasks version:** (see Nextcloud apps page)
??? Cloned on 5/25/17

**Updated from an older Nextcloud or fresh install:**
Upgraded from Owncloud 8.2 -> 9.0 -> 9.1.5

**Signing status:**

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
When app is trying to load, can't access owncloud.

Once I get owncloud working again by restarting apache2 and clearing maintenance mode, all I get is
Integrity checker has been disabled. Integrity cannot be verified.

**List of activated apps:**
Enabled:
  - activity: 2.3.2
  - calendar: true
  - comments: 0.3.0
  - configreport: 0.1.1
  - contacts: true
  - dav: 0.2.7
  - federatedfilesharing: 0.3.0
  - federation: 0.1.0
  - files: 1.5.1
  - files_antivirus: 0.9.0.0
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.10.0
  - files_texteditor: 2.1
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.0
  - notifications: 0.3.0
  - onlyoffice: 1.0.3
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - tasks: 0.8
  - templateeditor: 0.1
  - updatenotification: 0.2.1
Disabled:
  - documents
  - encryption
  - external
  - files_external
  - ownnote
  - user_external
  - user_ldap

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**
{
    "system": {
        "instanceid": "oc8pncae8r7f",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.mesozoictech.com"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/cloud.mesozoictech.com\/owncloud",
        "dbtype": "mysql",
        "version": "9.1.5.2",
        "dbname": "tf1_owncloud",
        "dbhost": "192.168.0.39:3306",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "theme": "",
        "loglevel": 0,
        "maintenance": true,
        "trashbin_retention_obligation": "auto",
        "updatechecker": false,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "preview_max_x": 2048,
        "preview_max_y": 2048,
        "preview_max_scale_factor": 10,
        "preview_max_filesize_image": 50,
        "preview_office_cl_parameters": " --headless --nologo --nofirststartwizard --invisible --norestore -convert-to pdf -outdir ",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown"
        ],
        "mail_from_address": "xxxxxx",
        "mail_smtpmode": "php",
        "mail_domain": "xxxxxxxxx"
    }
}

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your instance's installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
/var/www/html/owncloud/data is mapped to a FreeNAS share

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/nextcloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**
Version 58.0.3029.96 (64-bit)
**Operating system:**
Windows 10
**CalDAV-clients:**

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by raimund-schluessler** [comment]
Sorry to say, but this repository is only for Tasks on Nextcloud. The correct repository for Tasks on ownCloud is https://github.com/owncloud/tasks and there already is an issue for this problem (but I guess you already know). Since I don't use ownCloud anymore I cannot give support for any ownCloud related questions. Also, I do not maintain the ownCloud Tasks version anymore (there was a notice about that on the repository, but it was removed by the ownCloud guys). However, there has not been any change in the code of the app of which I would expect to break compatibility.

**Comment by raimund-schluessler** [comment]
Anyway, if you could post the ownCloud log, I am willing to take a look that happened. The message "an error occured" from the Webinterface is not very helpful ;)

Also, are you sure, you are using the latest Tasks version? In your list of activated apps it says "Tasks 0.8". The latest release is 0.9.5.

**Comment by tschiu** [comment]
Hi raimund.  Thank you for taking the time to look at this.  I hadn't seen the version 0.8, which is probably the basic problem.  I cloned the github repository from owncloud/tasks originally and *thought* I'd done the same with the repository here.  Maybe my clone is on the wrong branch...

I was going to request someone close this ticket as I'm migrating over to Nextcloud shortly (and the task app worked perfectly on test 11.03 instance I set up).
